zeroclickinfo-spice/lib/DDG/Spice/Catfacts.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Catfacts;
+
+# ABSTRACT: Gives random facts about cats
+
+use DDG::Spice;
+
+spice is_cached => 0;
+spice proxy_cache_valid => "418 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+spice to => 'http://catfacts-api.appspot.com/api/facts';
+
+triggers any => 'random cat fact', 'random cat facts', 'random facts about cats', 'random fact about cats';
+
+handle remainder => sub {
+
```

Comment:
We want ensure that the whole query is only our trigger, so we need to `return if $_;` meaning if the query contains anything else, don't show the Instant Answer.

Then you want to `return '';` to make an API call